src/sentry/tasks/collect_project_platforms.py

```diff
@@ -39,5 +39,5 @@ def collect_project_platforms(**kwargs):
 
     # remove (likely) unused platform associations
     ProjectPlatform.objects.filter(
-        last_seen__lte=now - timedelta(days=30),
+        last_seen__lte=now - timedelta(days=90),
```

Comment:
> I figured since this has been running forever, it wasn't hurting anything.

It's not horrible, but something like this is going to grow over time and will continue to get more and more expensive to run until it does cause a problem.